### ISSUE TITLE
Continuous integration: Build & deploy LaTeX documents

### ISSUE THREAD

**Issue by runphilrun** [RIT-Space-Exploration/SPEX-Standard-Proposal#11]
https://harshjv.github.io/blog/document-building-versioning-with-tex-document-git-continuous-integration-dropbox/

**Comment by venku122** [comment]
Looks cool. Google Drive doesn't have native webhooks like Dropbox does. It does have a REST api, so things will have to change. Lets plan on this as a mid semester weekend project for now.

**Comment by runphilrun** [comment]
I was just looking at a docker to auto-build PDD pdfs

**Comment by venku122** [comment]
Yeah, we can totally do that, but if we want to link to Google Drive, I'll need to do more research. 

Currently, just making people build and commit should be fine. Rarely should the .tex and pdf get out of sync.
